torch/csrc/Module.cpp

```diff
@@ -780,6 +780,9 @@ TORCH_API PyObject* initModule();
 // separate decl and defn for msvc error C2491
 PyObject* initModule() {
   HANDLE_TH_ERRORS
+
+  c10::initLogging();
+
```

Comment:
Although that would be a cleaner solution, there are two issues:

1) We still have to call `SetLogLevelFlagFromEnv` and several other functions when we use GLOG (see GLOG version of `initLogging` [here](https://github.com/pytorch/pytorch/pull/71746/files#diff-a662e72cf0e756a7e539c54f467cafa5f37d8a5e8eb8dd53959dadef54a2cd2cR226).

2) Since the logging unit is shared between PyTorch and caffe2, using a global constructor will interfere when used in caffe2 (which explicitly calls `initCaffe2Logging()`).